test/Microsoft.DotNet.ExecutablePackageObtainer.Tests/DotnetToolsConfigGolden.xml

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<DotnetToolMetadata>
+  <CommandName>sayhello</CommandName>
+  <ToolAssemblyEntryPoint>consoleappababab.dll</ToolAssemblyEntryPoint>

```

Comment:
Can we change the name of this dll? Maybe just consoleapp? Or some other name? Not a big fan of this randomly typed name on the code base.

Reply:
Will change, I thought I removed all random name

Reply:
Fixed